.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -3094,6 +3094,21 @@ stages:
           displayName: Publish release artifacts
           artifact: $(tracer_version)-release-artifacts
 
+        # lib-injection images
+        - bash: |
+            echo "ref is Build.SourceBranch=$(Build.SourceBranch)"
```

Comment:
1. I meant to delete this print statement as it was more of a debugging statement 😆 
2. It seems to work here and in other script tasks since this syntax resolves the variable before the task is executed. Since it's working now, I'm tempted to leave it as-is